### DISCUSSION TITLE
Advantage Database Server via ODBC

### DISCUSSION THREAD

**Discussion by lmaher007** [dbeaver/dbeaver#11708] [Q&A]
Not a feature request, but more of a question since I didn't find anywhere to ask simple quesitons.

I'm trying to use DBeaver on an ODBC connection to an Advantage Database Server (versions 8 or 9).

Simple question... Can it be done?

Thank you
Lynne

**Comment by uslss** [comment]
Hi @lmaher007 ,
did you try to use ODBC driver?
![image](https://user-images.githubusercontent.com/31996417/110532963-8aacef00-812e-11eb-98c1-2ea520f4ffa3.png)

**Comment by lmaher007** [comment]
Yes I did, but because our database is 32 bits, I had to install an older version of DBeaver in which I don't see functions and can't select tables on the left but I see stored procedures, but it seems to open the ODBC!!!

Oh and I cannot run a simple select on an existing table. One I know works. 

I guess DBeaver is a no go for me... sorry

**Comment by uslss** [comment]
Unfortunately, dbeaver supports x64 odbc driver only

**Comment by grlendistry** [comment]
Anyone tried to connect even with x64 odbc driver?

**Comment by kwisatz** [comment]
There is the added complexity now that the legacy ODBC driver has been removed from DBeaver CE 24.
I have downloaded the legacy driver, but honestly, the guides on how to set it up are incomplete or inexistant. 
Seems like the only viable option would be to use [the ODBC driver in the PRO version](https://dbeaver.com/docs/dbeaver/ODBC-JDBC-Driver/#installing-odbc-drivers), but…

If I understood this correctly, if the files in your data directory are from a 32-bit database, then using the 64-bit ODBC driver won't do me any good anyway?

See also #17569 17569
